FirebaseFunctionsSwift/Tests/Unit/FunctionsTests.swift

```diff
@@ -120,49 +120,30 @@ class FunctionsTests: XCTestCase {
 //  [super tearDown];
```

Comment:
Feel free to delete the old commented ObjC above. Otherwise I'll get it in my next PR.